bin/updateSourceModeration.ts

```diff
@@ -0,0 +1,32 @@
+import '../src/config';
+import createOrGetConnection from '../src/db';
+import { SquadSource } from '../src/entity';
+import { runInQueueStream } from './common';
+
+interface Result {
+  title: string;
+  id: string;
+}
+
+(async () => {
+  const con = await createOrGetConnection();
+  await con.transaction(async (manager) => {
+    console.log('initializing stream');
+
+    const builder = manager
+      .getRepository(SquadSource)
+      .createQueryBuilder('ss')
+      .select('ss.id', 'id')
+      .where('ss."memberPostingRank" = 0');
```

Comment:
Do we even need this though?
If the posting rank is 0 nobody but admins/mods can post anyway right?

Did it prove to be an issue somewhere?

Reply:
> Did it prove to be an issue somewhere?

Just saw this as part of the RFC. Maybe @ilasw can enlighten us with more details on why we need to update existing squads that allows posting for everyone?